core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/CoreDocumentSynchronizationConfig.java

```diff
@@ -127,12 +140,6 @@ public boolean isStale() {
   public void setStale(final boolean stale) {
     docLock.writeLock().lock();
     try {
-      docsColl.updateOne(
-          getDocFilter(namespace, documentId),
-          new BsonDocument("$set",
-              new BsonDocument(
-                  CoreDocumentSynchronizationConfig.ConfigCodec.Fields.IS_STALE,
-                  new BsonBoolean(stale))));
       isStale = stale;
```

Comment:
[q] approve of caching locally but looks like this never gets written to the docConfig as a result - are we ok with this?

Reply:
It does get written when we do the bulk writes.

In the event that the app crashes before it can be written, the document is marked stale anyway on app restart.